### ISSUE TITLE
Cannot access it from external network

### ISSUE THREAD

**Issue by sushruth** [markserv/markserv#1]
If you can not access your instance from external network, try running `markserv` from `0.0.0.0`. From the terminal, this:

``` bash
markserv -p 8642 -a 0.0.0.0 & # & - to run it without blocking your terminal instance
```

This is something that helped me so sharing it as an issue. 

Thanks to @F1LT3R  for this amazingly simple server! We did not have a wiki at my work so started using this as one!

**Comment by F1LT3R** [comment]
Great to hear you are finding it useful. Thanks for the tip!

Feel free to create PR adding that to the documentation if you wish.

**Comment by sushruth** [comment]
Sure! thanks.

**Comment by sushruth** [comment]
Solved in PR #2
